tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add new add-ons using maintenance test repo URLs.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub name_repo {
+    my ($repo_id, $addon) = $_[0] =~ (/(\d+)\/(.*)\//);
+    my $name = $addon . "_" . $repo_id;
+    return $name;   
+}
+
+sub run {
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    $testapi::distri->get_add_on_product()->confirm_like_additional_add_on();
+
+    my $maintrepo = shift @repos;
+    my $reponame = name_repo($maintrepo);
+    $testapi::distri->get_add_on_product()->accept_current_media_type_selection();
```

Comment:
could we please add a screenshot after this line to see every single screen in the test? just for completeness.

Reply:
Sure, done!